### ISSUE TITLE
[Feature/Improvement]: More of a question

### ISSUE THREAD

**Issue by Kaiji-Key** [Errored-Innovations/HeadsPlus#119]
### Checklist

- [X] Are you using the latest version/snapshot of [HeadsPlus](https://ci.pluginwiki.us/job/HeadsPlus/)?
- [X] Have you checked to see if others have submitted the same feature request?

### Description of the request

I have HeadsPlus set up on my Paper server, mainly for the head drop mechanic. I've looked at the config.yml file and I know that the default drop chance is 5, I assume 5/100, but heads seem to be dropping much more often than that, at more of a 15% rate. I wanted to see if this 5 does stand for 5/100 or not.

**Comment by Thatsmusic99** [comment]
Yep, it stands for 5%. There's a `/hp tests` subcommand that you can run for each mob type that runs a specified number of tests against the configured drop chance to see how often it passes.

5% is still 1 head for every 20 drops, so it's still semi-regular.

**Comment by Kaiji-Key** [comment]
Ah, I see. Thank you.

I do have another issue now though. In mobs.yml, I have tried a few times to change the drop odds for most of the mobs individually, but it keeps resulting in a mobs-errored-{mass of numbers).yml file appearing, and none of the odds being changed. I know I am doing something wrong, but I've tried everything I can think of. I've removed the {} in front of the HP#mob: and then adding the chance: 1, I've tried replacing the HP#mob: with '{mob-default}': as some of the mobs have, and I've tried having the HP#mob: followed by the '{mob-default}': and then the chance: 1. It's probably something really dumb that I'm missing and I'm sure my explanation doesn't make much sense either so please let me know if you want screenshots or something. Thanks!

**Comment by Thatsmusic99** [comment]
It's a syntax error in the YAML files, the console will often point out where the issue is. You can also use https://yamlchecker.com/ to verify the syntax of the file.
